funcbench/main.go

```diff
@@ -215,13 +220,14 @@ func startBenchmark(
 		return nil, errors.Wrap(err, "get head")
 	}
 
-	if _, err := bench.c.exec(ctx, "sh", "-c", "git update-index -q --ignore-submodules --refresh && git diff-files --quiet --ignore-submodules --"); err != nil {
+	// TODO move it into env? since GitHub env doesn't need this check.
```

Comment:
Why does not need? I think there is no harm, no?

Reply:
Because in GitHub env, the repo is always newly cloned, we presume that worktree is always clean.